docs/services/ml/upload-training-script.md

```diff
@@ -124,154 +307,45 @@ Dataset JSON files for image datasets with bounding box labels and classificatio
 
 {{% /expand%}}
-{{%expand "Step 3: Add logic to produce model artifact" %}}
-
-After reading in the dataset and preparing the data for training, add logic to build and compile the model using the data passed in through the data file.
-The code in this part of your script depends on the type of ML training you are doing.
+{{%expand "Logic to produce model artifact" %}}
 
-For example, you can refer to the logic from <file>model/training.py</file> from this [example classification training script](https://app.viam.com/packages/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb/custom-training-classification/ml_training/latest/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb) that trains a classification model using TensorFlow and Keras.
+The next part of the script depends on the type of ML training you are creating.
+In this part of the script, you use the data from the dataset and the annotations from the dataset file to build a Machine Learning model.
 
-The logic to build and compile the classification model looks like this:
-
-```python {class="line-numbers linkable-line-numbers"}
-# Build the Keras model
-def build_and_compile_classification(
-    labels: ty.List[str], model_type: str, input_shape: ty.Tuple[int, int, int]
-) -> Model:
-    units, activation, loss_fnc, metrics = get_neural_network_params(
-        len(labels), model_type
-    )
-
-    x = tf.keras.Input(input_shape, dtype=tf.uint8)
-    # Data processing
-    preprocessing = preprocessing_layers_classification(input_shape[:-1])
-    data_augmentation = tf.keras.Sequential(
-        [
-            tf.keras.layers.RandomFlip(),
-            tf.keras.layers.RandomRotation(0.1),
-            tf.keras.layers.RandomZoom(0.1),
-        ]
-    )
-
-    # Get the pre-trained model
-    base_model = tf.keras.applications.EfficientNetB0(
-        input_shape=input_shape, include_top=False, weights="imagenet"
-    )
-    base_model.trainable = False
-    # Add custom layers
-    global_pooling = tf.keras.layers.GlobalAveragePooling2D()
-    # Output layer
-    classification = tf.keras.layers.Dense(
-        units,
-        activation=activation,
-        name="output"
-    )
-
-    y = tf.keras.Sequential(
-        [
-            preprocessing,
-            data_augmentation,
-            base_model,
-            global_pooling,
-            classification,
-        ]
-    )(x)
-
-    model = tf.keras.Model(x, y)
-
-    model.compile(
-        loss=loss_fnc,
-        optimizer=tf.keras.optimizers.Adam(learning_rate=1e-3),
-        metrics=[metrics],
-    )
-    return model
-```
+As an example, you can refer to the logic from <file>model/training.py</file> from this [example classification training script](https://app.viam.com/packages/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb/custom-training-classification/ml_training/latest/e76d1b3b-0468-4efd-bb7f-fb1d2b352fcb) that trains a classification model using TensorFlow and Keras.
```

Comment:
I was going to make a follow-up ticket after the epic was completed, but I think we can just link to these two Github repos/registry items now:
- https://github.com/viam-modules/detection-tflite, https://app.viam.com/ml-training/viam/detection-tflite
- https://github.com/viam-modules/classification-tflite, https://app.viam.com/ml-training/viam/classification-tflite